### DISCUSSION TITLE
How to remove the firmware ?

### DISCUSSION THREAD

**Discussion by RomainD2** [roleoroleo/yi-hack-Allwinner#324] [Q&A]
Hi,

I tried this firmware on a Yi 1080p Home BFUS camera, but I have a lot of connection problem in RSTP mode, and ever with ONVIF server. I tried a lot of thing, the camera disconnect and I need to refresh my software to work again. I tried different software, same thing happened :(

Could you tell me how to come back to the original firmware ? I searched on the web, I found nothing :(

Thanks for your help !

**Comment by roleoroleo** [comment]
Check the wiki: unhack procedure.
